clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/subsystem/ThreadPoolResourceDefinition.java

```diff
@@ -61,10 +61,10 @@
  */
 public enum ThreadPoolResourceDefinition implements ResourceDefinition, Registration<ManagementResourceRegistration> {
 
-    DEFAULT("default", "thread_pool", 20, 300, 100, 60L),
-    OOB("oob", "oob_thread_pool", 20, 300, 0, 60L),
-    INTERNAL("internal", "internal_thread_pool", 2, 4, 100, 60L),
-    TIMER("timer", "timer", 2, 4, 500, 5L),
+    DEFAULT("default", "thread_pool", 20, 300, 100, 5000L),
+    OOB("oob", "oob_thread_pool", 20, 300, 0, 1000L),
+    INTERNAL("internal", "internal_thread_pool", 2, 4, 100, 60000L),
+    TIMER("timer", "timer", 2, 4, 500, 5000L),
```

Comment:
btw @pferraro so yes, we didnt define one earlier but this is 5 second the hardcoded default in the java file org.jgroups.protocols.TP#timer_keep_alive_time